packages/components/components/elvis-tabs/src/react/elvia-tabs.tsx

```diff
@@ -73,14 +85,16 @@ export const Tabs: FC<TabsProps> = ({
 
   return (
     <TabsContainer className={className} style={inlineStyle} data-testid="tabs-container" {...rest}>
-      <LeftButton
-        isVisible={['center', 'right'].includes(scrollPosition)}
-        size="md"
-        onClick={() => scroll('left')}
-        aria-label="Scroll mot venstre"
-      >
-        <IconWrapper icon={arrowLeftBold} size="xxs" color={isInverted ? 'white' : undefined} />
-      </LeftButton>
+      {scrollPosition !== 'left' && (
+        <LeftButton
+          isVisible={['center', 'right'].includes(scrollPosition)}
+          size="md"
+          onClick={() => scroll('left')}
+          aria-label={getScrollLabel('left')}
+        >
+          <IconWrapper icon={arrowLeftBold} size="xxs" color={isInverted ? 'white' : undefined} />
+        </LeftButton>
+      )}
```

Comment:
Why does this now need to be conditionally rendered, instead of how it used to be? 🤔

Reply:
We discussed briefly what solution would be better when the tabs component is used on smaller screens. We could simply keep the code as it was and just change the labels inline, but for readability and maintainability it made sense to keep the function that decides what labels to use separately. There is very little benefit of one over the other, but what I noticed is that both buttons are rendered even if they are not actually visible. When it comes to functionality both solutions are good, but with conditional rendering we only need to render the buttons we actually see along with their labels. As it is right now, we can see both left and right button-labels are rendered even though only one or none might be visible.